packages/web-components/fast-foundation/src/accordion/accordion.ts

```diff
@@ -125,25 +125,29 @@ export class Accordion extends FASTElement {
     }
 
     private handleItemKeyDown = (event: KeyboardEvent): void => {
-        const keyCode: number = event.keyCode;
-        this.accordionIds = this.getItemIds();
-        switch (keyCode) {
-            case keyCodeArrowUp:
-                event.preventDefault();
-                this.adjust(-1);
-                break;
-            case keyCodeArrowDown:
-                event.preventDefault();
-                this.adjust(1);
-                break;
-            case keyCodeHome:
-                this.activeItemIndex = 0;
-                this.focusItem();
-                break;
-            case keyCodeEnd:
-                this.activeItemIndex = this.accordionItems.length - 1;
-                this.focusItem();
-                break;
+        // only handle the keydown if the event target is the accordion item
+        // prevents arrow keys from moving focus to accordion headers when focus is on accordion item panel content
+        if (event.target === event.currentTarget) {
```

Comment:
nit: instead of wrapping the entire block, you could invert it and return when they're not equal.

Reply:
Yeah, would prefer a guard here rather than wrapping the entire block to execute. 

```
if (event.target !== event.currentTarget) {
    return;
}
```

Reply:
Updated!